internal/provisioner/kops_utils.go

```diff
@@ -191,3 +191,21 @@ func updateKopsInstanceGroupAMIs(kops *kops.Cmd, kopsMetadata *model.KopsMetadat
 
 	return nil
 }
+
+func updateKopsInstanceGroupInstanceMetadata(kops *kops.Cmd, kopsMetadata *model.KopsMetadata, logger log.FieldLogger) error {
```

Comment:
Can we change this function a bit? It would be nice if it was a general purpose function where I could pass any an instance group string value to `set` all the instance groups with. The calling function could then pass in `spec.instanceMetadata.httpTokens=optional` as an argument to set.